mpris2controller/controller.py

```diff
@@ -171,6 +172,12 @@ def call_on_head_not_playing(self, method_name):
             else:
                 self.remove(cur)
 
+    @dbus.service.method(dbus_interface=MY_INTERFACE)
+    def StopLoop(self):
+        log.info("Stop main loop")
```

Comment:
Just one log message before (like the other methods) should be sufficient :)

Reply:
Also, maybe this method should be called `Quit` on DBUS ("stop loop" seems implementation-specific, and just "stop" might be confused with MPRIS stop)